main.tex

```diff
@@ -135,25 +135,25 @@ \section*{Introduction}
 %
 The paths of transmission, carrying resistance alleles between mosquito populations, are also not known. 
 %
-Previous studies have found evidence that \texttt{L995F} occurs on several different genetic backgrounds, suggesting multiple independent outbreaks of resistance driven by this allele \cite{Pinto2007,Etang2009,Santolamazza2015}. 
+Previous studies have found evidence that \texttt{L995F} occurs on several different genetic backgrounds, suggesting multiple independent outbreaks of resistance driven by this allele \cite{Pinto2007,Etang2009,Lynd2010,Santolamazza2015}. 
 %
-However, these studies analysed only a small gene region in a limited number of mosquito populations, and therefore had limited resolution to make inferences about genetic relationships between gene sequences (haplotypes) carrying this allele. 
+However, these studies analysed only a small gene region (or several small regions in the case of \cite{Lynd2010}) in a limited number of mosquito populations, and therefore had limited resolution to make inferences about genetic relationships between gene sequences (haplotypes) carrying this allele. 
```

Comment:
Suggest to simplify this for readability as "...these studies analysed on small gene regions in a limited number of mosquito populations...".